src/components/BulkEnrollmentPage/stepper/constants.jsx

```diff
@@ -7,7 +7,13 @@ export const FINAL_BUTTON_TEXT = 'Enroll learners';
 export const NEXT_BUTTON_TEST_ID = 'stepper-next';
 export const PREV_BUTTON_TEST_ID = 'stepper-prev';
 export const FINAL_BUTTON_TEST_ID = 'stepper-final';
+export const NOTIFY_CHECKBOX_TEST_ID = 'notify-checkbox';
```

Comment:
Kudos: LOVE the test id convention we're using here, please call out and show to others.